lib/Doctrine/ORM/Tools/Console/Command/ValidateSchemaCommand.php

```diff
@@ -76,6 +77,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $ui->text('<comment>[SKIPPED] The database was not checked for synchronicity.</comment>');
         } elseif (! $validator->schemaInSyncWithMetadata()) {
             $ui->error('The database schema is not in sync with the current mapping file.');
+
+            if ($output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE) {
+                $updates = $validator->getUpdateSchemaList();
+
+                $rows = [];
+                foreach ($updates as $idx => $sql) {
+                    $rows[] = [(int) $idx + 1, $sql];
+                }
+
+                $ui->comment(sprintf('<info>%d</info> schema diff(s) detected:', count($updates)));
+                $ui->table(['n°', 'SQL'], $rows);
+            }
+
```

Comment:
> Can you give some hint about the tests that should be done?

I'd say here, let's not write a fragile test that would break every time Symfony changes the implementation of `$ui->table()`, but let's instead assert that the output contains the level of detail you expect (for instance, some table names are expected to be mentioned here, right?)